src/core/messages-reactions.ts

```diff
@@ -112,6 +113,34 @@ export interface MessagesReactions {
    * @returns A subscription object that should be used to unsubscribe.
    */
   subscribeRaw(listener: MessageRawReactionListener): Subscription;
+
+  /**
+   * Get the reaction count for a message for a particular client.
+   * @param messageSerial The serial of the message to remove the reaction from.
+   * @param clientId The client to fetch the reaction summary for (leave unset for current client).
+   * @returns A clipped reaction summary containing only the requested clientId.
+   * @example
+   * ```typescript
+   * // Subscribe to reaction summaries and check for specific client reactions
+   * room.messages.reactions.subscribe(async (event) => {
```

Comment:
Do `async` functions actually work here? I've never tried!

Reply:
Possible downside of async here is that you can't guarantee that this listener will be complete before the next message is received